apps/ledger-live-desktop/src/renderer/storage.ts

```diff
@@ -57,7 +57,7 @@ type DatabaseValues = {
   countervalues: Countervalues;
   postOnboarding: PostOnboarding;
   settings: Settings;
-  trustchainStore: TrustchainStore;
+  trustchain: TrustchainStore;
```

Comment:
```suggestion
  trustchain: Trustchain;
```

Reply:
not sure, it will conflict with our `Trustchain` type